components/FeedbackView/FeedbackView.tsx

```diff
@@ -63,36 +77,31 @@ const FeedbackView = ({
   };
 
   return (
-    <div className="flex flex-col m-4 h-full">
+    <div className="flex flex-col m-4 h-full" data-testid="feedback-view">
       {isLoading ? (
-        <>
-          <div className="flex justify-center items-center h-full">
-            <div className="border-t-4 border-red-500 border-solid w-12 h-12 rounded-full animate-spin" />
-          </div>
-        </>
+        <div
+          className="flex justify-center items-center h-full"
+          data-testid="feedback-view--loading"
+        >
+          <div className="border-t-4 border-red-500 border-solid w-12 h-12 rounded-full animate-spin" />
+        </div>
       ) : (
-        <>
+        <div
+          className="flex flex-col m-4 h-full"
+          data-testid="feedback-view--loaded"
```

Comment:
```suggestion
          data-testid="feedbackView__loading"
```
The `--` are for modifiers. This is a child element.